src/games/gameitem.py

```diff
@@ -298,58 +297,50 @@ def update(self, message, old_client=None):
         affectedplayers = oldplayers | newplayers
         client.instance.usersUpdated.emit(list(affectedplayers))
 
-    def editTooltip(self, teams):
-        
-        observerlist = []
+    def editTooltip(self, teams, observerlist):
+
         teamlist     = []
 
         teams_string = ""
+        for i, team in enumerate(teams):
```

Comment:
Adding to that previous comment - I checked out of interest, you actually can enumerate from something else than 0 if you give it the starting number as the second argument. Didn't even know that.